docs/cookbook-recipes/isle-v140-git-cleanup.md

```diff
@@ -0,0 +1,163 @@
+
+* a clean up script along with this documentation was created and run on the ISLE repository for the ISLE v.1.4.0 release.
+
+- Any new pull request will have to be created from a forked or cloned ISLE repository that has had this cleanup process performed on it.
+
+---
+
+## **AS OF THE ISLE v.1.4.0 release in November 2019** All ISLE endusers are asked to run this script following the criteria below in the "Do I need to run this script? section."
+
+---
+
+## Do I need to run this script?
+
+* If you are a new ISLE user and are cloning or forking this project for the first time, then **NO** you don't need to run the script below.
+
+- If you have previously forked or cloned the ISLE project **prior to** November 28, 2019, then **YES** you do need to run the script below.
+
+* If you still feel like you're not sure or cannot tell if you need to do this, please refer to the instructions below in the **Identification Tools** section.
+
+- If you intend to submit pull requests (PRs) of any kind (documentation, code etc) to the ISLE project, ISLE maintainers will **NOT** accept PRs from repos that haven't had the script run on them as using an uncleaned forked repo will cause the removed files to return.
+
+There is a brief description of what the script actually does found below in the **Script commands - explained** section.
+
+---
+
+## Git cleanup process
+
+
+### Git cleanup process - Overview
+
+The steps below in the section `Git cleanup process` will have you perform the following:
+
+* Open up a terminal and perform steps as directed
+- Make a directory called `ISLE_140_GIT_CLEANUP` on your laptop or workstation to perform all of these steps and collect all materials
+* Download the git cleanup script (`isle-v140-git-cleanup-script.sh`)and move it to the `ISLE_140_GIT_CLEANUP` directory
+- `git clone --mirror` your forked or cloned ISLE project directory to the `ISLE_140_GIT_CLEANUP` directory
+* Run the git cleanup script. It will prompt you to only enter the name of your forked or cloned ISLE project directory.
+- `git push` back the changes to your remote forked or cloned ISLE project directory.
+* Clone down your newly changed and smaller forked or cloned ISLE project directory again to ensure that the repositiory is smaller to a location of your choice.
+- Remove all previous larger size versions of your forked or cloned ISLE project directory. You can now delete the `ISLE_140_GIT_CLEANUP` directory and its contents.
+* Work from the newly resized and smaller forked or cloned ISLE project directory.
+  
+### Git cleanup process - Steps
+
+* Open up a terminal
+
+* Change directory to a location of your choice
+  * `cd your_path_here`
+
+* Make a new temporary direcotory called `ISLE_140_GIT_CLEANUP`
+    * `mkdir ISLE_140_GIT_CLEANUP`
+
+* Download the script found and save it to the `ISLE_140_GIT_CLEANUP` directory and fix the permissions.
+  * Right click on this link below and click "Save link as"
```

Comment:
This didn't work for me on a mac using Chrome.   We'll need another method.

Reply:
Maybe we need something using this: `https://raw.githubusercontent.com/...`

Reply:
Something like this, but I'm not sure if this will be the addr once this is accepted:  `wget 
 https://raw.githubusercontent.com/Islandora-Collaboration-Group/ISLE/0c1f7921d8c74f8ac0c4946f2672e51fb626e95e/docs/assets/isle-v140-git-cleanup-script.sh`.

Reply:
Ah, let me clarify - I'm seeing HTML when I do this and not the raw script itself.   Doing something like the `wget` above will be more consistent for folks using this.

Reply:
@bseeger Thanks for taking the time to chat re this. I see the same challenge for sure now. I've updated to use your example wget method above.